fullstack-projects/jwt-auth-fullstack/devashmit/backend/auth.py

```
@@ -0,0 +1,18 @@
+from datetime import datetime, timedelta, timezone
+from jose import JWTError, jwt
+from passlib.context import CryptContext
+
+SECRET = "change-me-in-production-long-random-secret"
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "SECRET"

(S105)